src/hooks/requests/useProjects.ts

```diff
@@ -193,6 +194,7 @@ export const useUpdateSubProject = ({ id }: { id?: string }) => {
           return { data: newData }
         }
       )
+      queryClient.refetchQueries({ queryKey: ['projects'] })
```

Comment:
**[blocker]**

Do we really need this ? we should avoid doing any unnecessary requests to BE. In most cases BE should return all the changed fields in the response of the PUT/POST request and we should be able to take the data from there and store it in our cache.

If something is missing, then first thing we should do is ask BE to add it and only if that is too much hassle for them, we can go with refetching.

**NB** Same comment applies for the refetch of subprojects after updating the project

Reply:
Thanks! Implemented 1.2 and 2.2 approaches, helped to nicely to refresh the view without having to refetch